src/main/java/com/wavefront/datastructures/Span.java

```diff
@@ -53,6 +54,23 @@ public Span(String name, long startMillis, long durationMillis, @Nullable String
     this.spanLogs = spanLogs;
```

Comment:
Code duplication. You can here call the next constructor with `..., SpanKind.REGULAR` value.